### PR TITLE
[REF] iap: adapt include on CrashManager

### PR DESCRIPTION
**Pull request by sdegueldre**

### CODE REVIEW

File: addons/iap/static/src/js/insufficient_credit_error_handler.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        trial: errorData.trial,
+    });
+    const content = $(QWeb.render("iap.redirect_to_odoo_credit", {
+        data: errorData,
+    }));
+    if (errorData.body) {
+        content.css("padding", 0);
+    }
+    const { _t } = env;
+    const isEnterprise = _.last(odoo.session_info.server_version_info) === "e";
```
Comment (aab-odoo) [inline addons/iap/static/src/js/insufficient_credit_error_handler.js]: maybe use odoo.info?

File: addons/iap/__manifest__.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
     'assets': {
         'web.assets_backend': [
             'iap/static/src/js/**/*',
+            'iap/static/src/xml/*.xml',
```
Comment (aab-odoo) [inline addons/iap/__manifest__.py]: ? templates are in 'web.assets_qweb' bundle

File: addons/iap/static/src/js/insufficient_credit_error_handler.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        const { errorData } = this.props;
+        this.url = await this.orm.call("iap.account", "get_credits_url", [], {
+            base_url: errorData.base_url,
+            service_name: errorData.service_name,
+            credit: errorData.credit,
+            trial: errorData.trial,
+        });
+        this.style = errorData.body ? "padding:0;" : "";
+        const { _t } = this.env;
+        const isEnterprise = _.last(odoo.info.server_version_info) === "e";
```
Comment (aab-odoo) [inline addons/iap/static/src/js/insufficient_credit_error_handler.js]: add odoo.info.isEnterprise flag?

**Review by aab-odoo — APPROVED** [review]

**Comment by aab-odoo** [comment]
robodoo rebase-ff r+

**Comment by sdegueldre** [comment]
@robodoo retry

**Pull request merged**